src/App.js

```diff
@@ -1,62 +1,69 @@
 import React, {Component} from 'react';
-import TodoList from './components/TodoComponents/TodoList';
-
-let storage = window.localStorage;
-let storageList = [];
-if (storage.list !== undefined){
-	storageList = JSON.parse(storage.list);
-}
+import Form from './components/TodoComponents/TodoForm'
+import List from './components/TodoComponents/TodoList'
+import './App.css'
 
 class App extends React.Component {
-constructor() {
-  super();
-  this.state = {todo: storageList, current: ''}
-}
-
-handleInputChange = event => {
-  this.setState({current: event.target.value});
-}
+  constructor(props){
+    super(props);
+    this.state={
+      input: '',
+     
+      todolist: [
+        {
+          task: 'graduate lambda school',
+          id: 1,
+          completed: false,
+        }
 
-addTodo = (event, prevState) => {
-  event.preventDefault();
-  let clear = document.querySelector('.toDoForm input');
-  clear.value = '';
-  let newList = this.state.todo.slice();
-  let newItem = {task: this.state.current, id: Date.now(), completed: false};
-  newList.push(newItem);
-  this.setState({todo: newList, current: ''});
-  storage.setItem('list', JSON.stringify(newList));
-}
+      ],
+     
+    };
+  }
 
+  handleInput = (event) => {
+    const {value} = event.target;
+    this.setState({
+      input: value,
+    });
+  };
 
-completeTask = event => {
-		let newList = this.state.todo.slice();
-		for (let todo in newList){
-			if (newList[todo].task === event.target.innerText && newList[todo].id === Number(event.target.dataset.id)){
-				newList[todo].completed = !newList[todo].completed;
-			}
-		}
-		this.setState({todo: newList});
-		storage.setItem('list', JSON.stringify(newList));
-	}
-
-removeCompleted = () => {
-  let currentList = this.state.todo.slice();
-  let newList = currentList.filter(todo => todo.completed === false);
-  this.setState({todo: newList});
-  storage.setItem('list', JSON.stringify(newList));
-}
+  handleClick = (id) => {
+    let list = this.state.todolist.slice();
+    let todo = list.findIndex(todo => todo.id === id);
+    list[todo].completed = true
```

Comment:
you could change this to toggle between true and false with `list[todo].completed = !list[todo].completed`